lib/scss/components/event-card/stanford_events.event-card.scss

```diff
@@ -57,3 +57,25 @@
     }
   }
 }
+
+// Single Event card per row on screens less than and equal to md.
+// For the single per paragraph row.
+.react-paragraphs-row {
```

Comment:
`.react-paragraphs-row` is not part of the events module. Any  of those styles related to the events stuff should live in the stanford_profile_helper.

Reply:
fixing.